oz/error_pages/__init__.py

```diff
@@ -41,7 +40,8 @@ def get_lines_from_file(filename, lineno, context_lines):
     Returns (pre_context_lineno, pre_context, context_line, post_context).
     """
 
-    get_lines = lambda s, e: [linecache.getline(filename, l).rstrip() for l in range(s, e)]
```

Comment:
Actually, lambda expressions in cases like this are not preferred by PEP8:

["Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier."](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)